css-four-column-layout/styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  background-color: #f8f8fa;
+}
+
+h1,
+h2,
+p {
+  font-family: 'Barlow', sans-serif;
+}
+
+h1 {
+  font-weight: 700;
+}
+
+h2 {
+  font-size: 1.2rem;
+  font-weight: bolder;
+}
+
+.container {
+  width: 75%;
+  margin: auto;
+}
+
+.row-header {
+  text-align: center;
+}
```

Comment:
You already have a class that does this below named `text-center`, you should remove this.